common/validator_dir/tests/tests.rs

```diff
@@ -78,13 +78,11 @@ impl Harness {
          * Build the `ValidatorDir`.
          */
 
-        let builder = Builder::new(
-            self.validators_dir.path().into(),
-            self.password_dir.path().into(),
-        )
-        // Note: setting the withdrawal keystore here ensure that it can get overriden by later
-        // calls to `random_withdrawal_keystore`.
-        .store_withdrawal_keystore(config.store_withdrawal_keystore);
+        let builder = Builder::new(self.validators_dir.path().into())
+            .password_dir(self.password_dir.path())
+            // Note: setting the withdrawal keystore here ensure that it can get overriden by later
```

Comment:
"ensures that it can get overridden".  Also not totally sure if this should be "can" or "can't"